charts/fluent-operator/templates/fluentbit-output-loki.yaml

```diff
@@ -108,7 +108,8 @@ spec:
 {{ .removeKeys | toYaml | indent 6 }}
     {{ end -}}
     
-    {{ if .tls -}}
+    {{ if .tls }}
```

Comment:
@wanjunlei @wenchajun should we keep the `-` here like `{{ if .tls -}}`?

Reply:
Just for the purpose of indentation and spacing, so I think it can be modified like this here.